index.js

```diff
@@ -12,8 +17,15 @@ function init() {
 
 	const selectedText = editor.getSelectedText();
 	const text = selectedText || editor.getText();
+	const config = atom.config.get('perfectionist');
 
-	postcss(perfectionist(atom.config.get('perfectionist'))).process(text, {
+	if (editor.getGrammar().scopeName === 'source.css.scss') {
```

Comment:
@rwwagner90 SCSS is a superset of CSS, so all CSS works in the SCSS parser, but not the other way around, by using SCSS on unsaved buffers we make it work for both CSS and SCSS.